### PR TITLE
Issues 73 - Add 'what does wikipedia say about' to wiki skill vocab

### PR DESCRIPTION
**Pull request by ethanaward**
I added 'what does wikipedia say about' to both the vocab file for the wiki skill and to prefixes in the __init__.py file for the wiki skill so that it can be used to trigger a wikipedia search.

### CODE REVIEW

**Comment by ryanleesipes** [comment]
While we are messing with this, please remove 'who is' and 'who was' from the vocab. We are going to let Wolfram Alpha handle that.

**Comment by ryanleesipes** [comment]
Oh no! These are failing @ethanaward - run ./start.sh unittest to do this locally, so you know whether or not it will fail before pushing.

**Comment by ryanleesipes** [comment]
@jdorleans any idea why this is failing? I can literally see why it is failing, but I don't know how to solve it.

File: mycroft/skills/wiki/__init__.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def initialize(self):
         self.load_vocab_files(join(dirname(__file__), 'vocab', 'en-us'))
 
         prefixes = ['wiki', 'wikipedia', 'tell me about', 'tell us about',
-                    'who is', 'who was']  # TODO - i10n
+                    'who is', 'who was', 'what does wikipedia say about']  # TODO - i10n
```
Comment (jdorleans) [inline mycroft/skills/wiki/__init__.py]:
```
mycroft/skills/wiki/__init__.py:52:80: E501 line too long (88 > 79 characters)
The command "pep8 mycroft test" exited with 1.
```

**Comment by jdorleans** [comment]
Personally, I think 80 characters is **too** restrict now days. We get to adjust those PEP8. For the time begin, please, fix it.

**Pull request closed without merging**